2018-10-02T17:39:33.884-0500 I CONTROL  [initandlisten] MongoDB starting : pid=22445 port=27017 dbpath=/data/db 64-bit host=installminsMBP5
2018-10-02T17:39:33.885-0500 I CONTROL  [initandlisten] db version v4.0.0
2018-10-02T17:39:33.885-0500 I CONTROL  [initandlisten] git version: 3b07af3d4f471ae89e8186d33bbb1d5259597d51
2018-10-02T17:39:33.885-0500 I CONTROL  [initandlisten] allocator: system
2018-10-02T17:39:33.885-0500 I CONTROL  [initandlisten] modules: none
2018-10-02T17:39:33.885-0500 I CONTROL  [initandlisten] build environment:
2018-10-02T17:39:33.885-0500 I CONTROL  [initandlisten]     distarch: x86_64
2018-10-02T17:39:33.885-0500 I CONTROL  [initandlisten]     target_arch: x86_64
2018-10-02T17:39:33.885-0500 I CONTROL  [initandlisten] options: { processManagement: { fork: true }, systemLog: { destination: "file", path: "../../bash/vocab/mongo_log/mongod.log" } }
2018-10-02T17:39:33.888-0500 I STORAGE  [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2018-10-02T17:39:33.888-0500 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7680M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2018-10-02T17:39:34.740-0500 I STORAGE  [initandlisten] WiredTiger message [1538519974:740906][22445:0x7fffc99ae3c0], txn-recover: Main recovery loop: starting at 60/53248
2018-10-02T17:39:34.836-0500 I STORAGE  [initandlisten] WiredTiger message [1538519974:836822][22445:0x7fffc99ae3c0], txn-recover: Recovering log 60 through 61
2018-10-02T17:39:34.911-0500 I STORAGE  [initandlisten] WiredTiger message [1538519974:911476][22445:0x7fffc99ae3c0], txn-recover: Recovering log 61 through 61
2018-10-02T17:39:34.960-0500 I STORAGE  [initandlisten] WiredTiger message [1538519974:960881][22445:0x7fffc99ae3c0], txn-recover: Set global recovery timestamp: 0
2018-10-02T17:39:35.074-0500 I RECOVERY [initandlisten] WiredTiger recoveryTimestamp. Ts: Timestamp(0, 0)
2018-10-02T17:39:35.391-0500 I CONTROL  [initandlisten] 
2018-10-02T17:39:35.392-0500 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2018-10-02T17:39:35.392-0500 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2018-10-02T17:39:35.392-0500 I CONTROL  [initandlisten] 
2018-10-02T17:39:35.393-0500 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2018-10-02T17:39:35.393-0500 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2018-10-02T17:39:35.394-0500 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2018-10-02T17:39:35.394-0500 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2018-10-02T17:39:35.394-0500 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2018-10-02T17:39:35.395-0500 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2018-10-02T17:39:35.395-0500 I CONTROL  [initandlisten] 
2018-10-02T17:39:35.570-0500 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2018-10-02T17:39:35.572-0500 I NETWORK  [initandlisten] waiting for connections on port 27017
2018-10-02T17:39:36.223-0500 I NETWORK  [listener] connection accepted from 127.0.0.1:51552 #1 (1 connection now open)
2018-10-02T17:39:36.224-0500 I NETWORK  [conn1] received client metadata from 127.0.0.1:51552 conn1: { driver: { name: "PyMongo", version: "3.7.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.5" }, platform: "CPython 3.6.3.final.0" }
2018-10-02T17:39:36.759-0500 I NETWORK  [listener] connection accepted from 127.0.0.1:51557 #2 (2 connections now open)
2018-10-02T17:39:36.760-0500 I NETWORK  [conn2] received client metadata from 127.0.0.1:51557 conn2: { driver: { name: "PyMongo", version: "3.7.1" }, os: { type: "Darwin", name: "Darwin", architecture: "x86_64", version: "10.12.5" }, platform: "CPython 3.6.3.final.0" }
2018-10-02T17:42:38.132-0500 I NETWORK  [conn2] end connection 127.0.0.1:51557 (1 connection now open)
2018-10-02T17:42:38.573-0500 I NETWORK  [conn1] end connection 127.0.0.1:51552 (0 connections now open)
2018-10-02T17:42:38.617-0500 I NETWORK  [listener] connection accepted from 127.0.0.1:52301 #3 (1 connection now open)
2018-10-02T17:42:38.619-0500 I NETWORK  [listener] connection accepted from 127.0.0.1:52303 #4 (2 connections now open)
2018-10-02T17:42:38.621-0500 I NETWORK  [listener] connection accepted from 127.0.0.1:52305 #5 (3 connections now open)
2018-10-02T17:42:38.623-0500 I NETWORK  [conn3] end connection 127.0.0.1:52301 (2 connections now open)
2018-10-02T17:42:38.623-0500 I NETWORK  [conn4] end connection 127.0.0.1:52303 (1 connection now open)
2018-10-02T17:42:38.623-0500 I NETWORK  [conn5] end connection 127.0.0.1:52305 (0 connections now open)
2018-10-02T17:42:38.641-0500 I NETWORK  [listener] connection accepted from 127.0.0.1:52309 #6 (1 connection now open)
2018-10-02T17:42:38.643-0500 I NETWORK  [listener] connection accepted from 127.0.0.1:52311 #7 (2 connections now open)
2018-10-02T17:42:38.646-0500 I NETWORK  [listener] connection accepted from 127.0.0.1:52313 #8 (3 connections now open)
2018-10-02T17:42:38.647-0500 I NETWORK  [conn6] end connection 127.0.0.1:52309 (2 connections now open)
2018-10-02T17:42:38.647-0500 I NETWORK  [conn7] end connection 127.0.0.1:52311 (1 connection now open)
2018-10-02T17:42:38.647-0500 I NETWORK  [conn8] end connection 127.0.0.1:52313 (0 connections now open)
2018-10-02T17:42:38.664-0500 I NETWORK  [listener] connection accepted from 127.0.0.1:52317 #9 (1 connection now open)
2018-10-02T17:42:38.666-0500 I NETWORK  [listener] connection accepted from 127.0.0.1:52319 #10 (2 connections now open)
2018-10-02T17:42:38.669-0500 I NETWORK  [conn9] end connection 127.0.0.1:52317 (1 connection now open)
2018-10-02T17:42:38.669-0500 I NETWORK  [conn10] end connection 127.0.0.1:52319 (0 connections now open)
2018-10-02T17:42:39.282-0500 I CONTROL  [signalProcessingThread] got signal 15 (Terminated: 15), will terminate after current cmd ends
2018-10-02T17:42:39.282-0500 I NETWORK  [signalProcessingThread] shutdown: going to close listening sockets...
2018-10-02T17:42:39.282-0500 I NETWORK  [signalProcessingThread] removing socket file: /tmp/mongodb-27017.sock
2018-10-02T17:42:39.283-0500 I CONTROL  [signalProcessingThread] Shutting down free monitoring
2018-10-02T17:42:39.283-0500 I FTDC     [signalProcessingThread] Shutting down full-time diagnostic data capture
2018-10-02T17:42:39.285-0500 I STORAGE  [signalProcessingThread] WiredTigerKVEngine shutting down
2018-10-02T17:42:39.446-0500 I STORAGE  [signalProcessingThread] shutdown: removing fs lock...
2018-10-02T17:42:39.448-0500 I CONTROL  [signalProcessingThread] now exiting
2018-10-02T17:42:39.448-0500 I CONTROL  [signalProcessingThread] shutting down with code:0
